src/resource_server/views/api_v1/api-spec/paths/index.yml

```diff
@@ -0,0 +1,8 @@
+/slurm:
+  $ref: ./slurm/index.yml
+
+/vnc:
+  $ref: ./vnc/index.yml
+
+/group:
```

Comment:
are there other categories of commands under `/group` `/vnc` compared to `/slurm` ?